gloo/test/CMakeLists.txt

```diff
@@ -5,17 +5,24 @@ set(GLOO_TEST_SRCS
   "${CMAKE_CURRENT_SOURCE_DIR}/barrier_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/broadcast_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/gather_test.cc"
-  "${CMAKE_CURRENT_SOURCE_DIR}/linux_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/main.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/memory_test.cc"
-  "${CMAKE_CURRENT_SOURCE_DIR}/multiproc_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/reduce_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/send_recv_test.cc"
-  "${CMAKE_CURRENT_SOURCE_DIR}/transport_test.cc"
   )
+set(GLOO_TEST_LIBRARIES)
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+  list(APPEND GLOO_TEST_SRCS
+    "${CMAKE_CURRENT_SOURCE_DIR}/linux_test.cc"
+    "${CMAKE_CURRENT_SOURCE_DIR}/multiproc_test.cc"
+    "${CMAKE_CURRENT_SOURCE_DIR}/transport_test.cc"
+    )
+  list(APPEND GLOO_TEST_LIBRARIES rt)
```

Comment:
what is `rt` here?

Reply:
`librt` contains functions including `clock_gettime`. This is used in `multiproc_test.cc`.